.eslintrc.js

```diff
@@ -61,7 +61,8 @@ module.exports = {
         'no-var': ['error'],
         'space-before-blocks': ['error'],
         'space-infix-ops': 'error',
-        'yoda': 'error'
+        'yoda': 'error',
+        'no-sequences': ['error']
```

Comment:
More errors will be found.
```suggestion
        'no-sequences': ['error', { 'allowInParentheses': false }]
```

Reply:
I had missed that @dmitrylyzo had already made suggestions here for the remaining issues found by eslint.